account_analytic_project/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================================
+Project and analytic account integration
+========================================
+
+Gives the option to create projects directly from the analytic accounts they
+should be linked to.
+
+Without this module, if you first create an analytic account and later on you
+decide to have a project as well for this account, the situation is very
+difficult to rectify.
+
```

Comment:
This is not like you say. It's very easy to create a project from an analytic line that already exists. I don't see any sense to this module, could you explain me better?

- Go to runbot
- Go to Accounting - Configuration -> Analytic Accounting -> Analytic account
- Create a new Analytic account
![2015-12-16_10-08-32](https://cloud.githubusercontent.com/assets/8736623/11836771/19094e22-a3dd-11e5-903f-946e138d92c5.png)
- Go to your analytic account again
- Enable "task", Odoo core will generate automatically a project
![2015-12-16_10-10-15](https://cloud.githubusercontent.com/assets/8736623/11836820/6489f720-a3dd-11e5-9f27-0796082a1c9c.png)

In my side :-1: , maybe I miss understood something?

Thanks